### ISSUE TITLE
Need to eliminate need to set #rowanCompile to true to enable a GemStone user to use Jadeite

### ISSUE THREAD

**Issue by dalehenrich** [dalehenrich/Rowan#177]
... we decided not to make this change for Alpha2.0, but it should be addressed before next drop

**Comment by dalehenrich** [comment]
... it just so happens that I ran into a case earlier today (or yesterday) where I did "need to compile an unpackaged method", so it probably means that an escape hatch is probably necessary ... which is why we deferred making the change in the first place:)

**Comment by ericwinger** [comment]
Yeah, I need to keep this around as well. At least until the JadeServer hierarchy is brought into Rowan.
